gst/nnstreamer/tensor_if/gsttensorif.c

```diff
@@ -231,8 +293,53 @@ static void
 gst_tensor_if_set_property (GObject * object, guint prop_id,
     const GValue * value, GParamSpec * pspec)
 {
-  /* GstTensorIf *self = GST_TENSOR_IF (object); */
-  /** @todo NYI! */
+  GstTensorIf *self = GST_TENSOR_IF (object);
+
+  g_critical ("Setting property for prop %d.\n", prop_id);
```

Comment:
CRITICAL == ERROR?

Reply:
This was added for debugging.. I removed this line. Thank you.